packages/navigator/src/browser/open-editors-widget/open-editors.css

```diff
@@ -0,0 +1,84 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+:root {
+    --theia-open-editors-icon-width: 16px;
+}
+
+.theia-open-editors-widget .theia-caption-suffix {
+    margin-left: var(--theia-ui-padding);
+    font-size: var(--theia-ui-font-size0);
+}
+
+.open-editors-node-row .open-editors-prefix-icon-container {
```

Comment:
I believe we should prefix the new stylings with `theia-open-editors-widget`

Reply:
👌